spec/unit/pops/types/enumeration_spec.rb

```diff
@@ -1,50 +1,51 @@
 require 'spec_helper'
 require 'puppet/pops'
 
+module Puppet::Pops::Types
```

Comment:
Because all of the stuff rspec /mocha is doing - it now does that inside of the module under test; while that can be debated if it is right or wrong philosophically, I am nervous there is stuff left behind in the module with the possible negative effect on other tests of things in the very same module.